app/client/service_runner.py

```diff
@@ -88,7 +85,7 @@ def is_queue_empty():
             raise Exception('Master service did not become idle before timeout.')
 
     def kill(self):
-        self._run_service('{} stop'.format(self._main_executable))
+        self._run_service([self._main_executable, 'stop'])
 
     def _run_service(self, cmd, service_url=None):
```

Comment:
No, I was asking whether it was okay to pass in a list of strings, instead of a string as was there previously.